src/flask/typing.py

```diff
@@ -33,15 +33,21 @@
     "WSGIApplication",
 ]
 
+GenericException = t.TypeVar("GenericException", bound=Exception, contravariant=True)
```

Comment:
@davidism I vaguely remember we had performance concerns about Generics, which I think doesn't apply here. Could you confirm this is fine?

Reply:
This isn't a `Generic`, so it's fine in that regard. I have no idea what this type does though.